app/Controllers/CtrlPublicPages.php

```diff
@@ -15,6 +15,8 @@
 
 class CtrlPublicPages extends BaseController
 {
+    private const CERTIFICATES_PER_PAGE = 9;
```

Comment:
Since we already have 4 columns on big screens change this value to just 10

```suggestion
    private const CERTIFICATES_PER_PAGE = 10;
```